src/WakaTime.Shared.ExtensionUtils/WakaTime.cs

```diff
@@ -91,17 +93,19 @@ public void HandleActivity(string currentFile, bool isWrite, string project)
             _lastFile = currentFile;
             _lastHeartbeat = now;
 
-            AppendHeartbeat(currentFile, isWrite, now, project);
+            AppendHeartbeat(currentFile, isWrite, now, project, activity, entityType);
         }
 
-        private void AppendHeartbeat(string fileName, bool isWrite, DateTime time, string project)
+        private void AppendHeartbeat(string fileName, bool isWrite, DateTime time, string project, HeartbeatCategory category, EntityType entityType)
         {
             var h = new Heartbeat
             {
                 Entity = fileName,
                 Timestamp = ToUnixEpoch(time),
                 IsWrite = isWrite,
-                Project = project
+                Project = project,
+                Category = category,
+                EntityType = entityType
             };
             HeartbeatQueue.Enqueue(h);
```

Comment:
Insert a break line here please.

Reply:
You mean before or after `HeartbeatQueue.Enqueue(h);`?